projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/address-book.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { editedAddress, newAddress } from '../../helpers/address-book';
+import { fillShippingAddress } from '../../helpers/checkout-forms';
+
+describe('Address Book Page Accessibility', { testIsolation: false }, () => {
+  before(() => {
+    cy.a11yContinuumSetup();
+    cy.requireLoggedIn();
+    cy.visit('/');
+  });
+
+  it('Empty page', () => {
+    cy.visit('/my-account/address-book');
+
+    cy.get('cx-address-form').should('exist');
```

Comment:
```suggestion
    cy.get('cx-address-form');
```
All extra assertions are unnecessary.

Reply:
Just updated.